src/moon_buggy_msgs/msg/ThrottleSteerCmd.msg

```diff
@@ -0,0 +1,8 @@
+# Header containing timestamp
+Header header
+
+# Throttle Command [1.0: full forward, -1.0: full reverse]
+float64 throttle_cmd
+
+# Steering Command [1.0: full left, -1.0: full right]
+float64 steering_cmd
```

Comment:
Are the values place holders?

Reply:
This is just defining the message type. The comments are just for documentation purpose to define what the float means. Ultimately it will need to get mapped to a uint8 on the teensy, but I figured this would be easier to work with